CodeForTucson/codefortucson-site
# Fix rendering issues left-over from calendar

In #41 we removed the duplicate calendar page, but this inadvertently
produced a rendering error on the home page - a big green block where it
should have been white. This error was the result of a failed page
generation.

**Wrong**
![wrong-1200](https://cloud.githubusercontent.com/assets/5431237/10334741/06b57c8c-6ca1-11e5-9381-5da2ab03e934.png)

**Right**
![right-1200](https://cloud.githubusercontent.com/assets/5431237/10334766/3a77b832-6ca1-11e5-971b-6b74edb9c110.png)

First of all, there was an error in the `navbar.html` template which I
have removed, though this was unrelated to the render issue.

Secondly, and more importantly, I believe that we ran into this issue
because there were two `index.html` files with overlapping slugs. After
removing the `index.html` file in the root directory, the rendering
glitch is gone.

**To Test**
Please build and navigate around to the various pages and make sure nothing is out of date. I literally wiped out the root `index.html` and think it was the erroneous one, but I'm not 100% sure.
